.github/workflows/static-analysis.yml

```diff
@@ -24,3 +24,11 @@ jobs:
       if: ${{ always() }} # Upload even if the previous step failed.
       with:
         sarif_file: build/reports/detekt/merged.sarif
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+    - name: Check REUSE Compliance
+      run: |
+        pip install --user reuse
+        ~/.local/bin/reuse lint
```

Comment:
The job name "Detekt" doesn't fit anymore. I'd suggest to run this in a separate action, because it's nice that from a single look at the checks you already know where the issue is.
Or did you just forget to add this to another job in the same file? Would they show separately in the GitHub UI then?

Reply:
Yes, this should have been in a separate job in the same file, fixed. See the check in this PR fro how it's displayed: Individual jobs show up in separate lines.